app-editors/scite/scite-4.3.0.ebuild

```diff
@@ -0,0 +1,113 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit toolchain-funcs xdg-utils
+
+MY_PV=${PV//./}
```

Comment:
it is only used once in the ebuild, so just do `${PV//./}` there instead of using a var.